InventoryItem.cpp

```diff
@@ -2,11 +2,7 @@
 #include "InventoryItem.h"
 
 
-InventoryItem::InventoryItem()
-{
-}
+InventoryItem::InventoryItem() {}
```

Comment:
This one doesn't matter too much, but considering that you put the opening brace on the same line, it seems weird to put the closing one on its own line when you have no contents to speak of, unless you had a comment in there of some sort. This is the common way I've seen to show an empty definition.